api/core/tools/provider/builtin/jina/tools/jina_reader.py

```diff
@@ -26,6 +27,9 @@ def _invoke(self,
         if 'api_key' in self.runtime.credentials and self.runtime.credentials.get('api_key'):
             headers['Authorization'] = "Bearer " + self.runtime.credentials.get('api_key')
 
+        request_params = tool_parameters.get('request_params', None)
+        request_params = json.loads(request_params) if request_params is not None and request_params != '' else None
```

Comment:
We should do a check here to make sure this is a valid json and make sure the notice of this error should be clear.